src/routes/devRoute.js

```diff
@@ -13,6 +13,7 @@ const {
   updateUserScorePhase2,
   removeActivity
 } = require('../services/devHandler');
+const newsProcessDevProcess = require('../services/devHandler/newsProcessDevProcess');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `newsProcessDevProcess` is inconsistent with the other imports from the same directory. For better readability and maintainability, it's recommended to use destructuring assignment to import multiple exports from a single module.
```diff
- const newsProcessDevProcess = require('../services/devHandler/newsProcessDevProcess');
+ const { newsProcessDevProcess } = require('../services/devHandler');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
will change

Reply:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

@fajarism That sounds great! Using destructuring for imports not only improves readability but also makes the code more consistent. Here's how you can change it:

```diff
- const {
-   updateUserScorePhase2,
-   removeActivity
- } = require('../services/devHandler');
- const newsProcessDevProcess = require('../services/devHandler/newsProcessDevProcess');

+ const {
+   updateUserScorePhase2,
+   removeActivity,
+   newsProcessDevProcess
+ } = require('../services/devHandler');
```

This way, all the functions from `devHandler` are imported in the same statement, which is cleaner and easier to maintain.

<!-- This is an auto-generated reply by OSS CodeRabbit -->